### DISCUSSION TITLE
Processing cosmo skymed data with gmtsar

### DISCUSSION THREAD

**Discussion by kapusuzcagri** [gmtsar/gmtsar#614] [General]
I am a senior student at Akdeniz University, Department of Space Sciences and Technologies. Within the scope of my thesis, I need to make deformation analysis by processing cosmo skymed data with gmtsar software, I scanned many files for this and I could not reach a conclusion about how to process it, can you support me?

**Comment by Xiaohua-Eric-Xu** [comment]
You could use p2p_processing.csh to create pairs of interferograms and then form time-series. If you would like to run batch processing, you could provide us with 3 acquisitions of data and I could get you a simple script to run that.
